packages/commercetools/composables/src/useCheckout/createPlaceOrder.ts

```diff
@@ -9,12 +9,13 @@ const createPlaceOrder = ({ factoryParams, cartFields, setCart }) => async () =>
   const { id, version } = cartFields.cart.value;
 
   const orderResponse = await createMyOrderFromCart({ id, version });
-  const cartResponse = await createCart();
-  setCart(cartResponse.data.cart);
-  initFields(cartResponse.data.cart);
+  const { order } = orderResponse.data;
+
+  const cartResponse = await deleteCart(order.cart.id, order.cart.version);
+  setCart(null);
```

Comment:
When placing an order from lets say order review page, you will call `await placeOrder()` after which you would want to redirect someone to lets say success page, but, when that is done, your current page will throw errors in different components saying it cannot read its data because cart is `null` and before the redirect your page will flash because it will become empty for a moment because the cart is `null`